src/main/java/uk/gov/companieshouse/items/orders/api/dto/CertificateItemDTO.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.companieshouse.items.orders.api.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import uk.gov.companieshouse.items.orders.api.model.CertificateItemOptions;
+
+/**
+ * An instance of this represents the JSON serializable certificate item for use in REST requests and responses.
+ */
+@JsonPropertyOrder(alphabetic = true)
+public class CertificateItemDTO extends ItemDTO {
+
+    @JsonProperty("item_options")
+    private CertificateItemOptions itemOptions;
+
+    public CertificateItemOptions getItemOptions() {
+        return itemOptions;
+    }
+
+    public void setItemOptions(CertificateItemOptions itemOptions) {
+        this.itemOptions = itemOptions;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
Can use Gson library for toString instead;

` @Override
    public String toString() {
        return new Gson().toJson(this);
    }`